jena-arq/src/main/java/org/apache/jena/sparql/syntax/syntaxtransform/ExprTransformApplyElementTransform.java

```diff
@@ -43,7 +43,7 @@ public ExprTransformApplyElementTransform(ElementTransform transform, boolean al
     @Override
     public Expr transform(ExprFunctionOp funcOp, ExprList args, Op opArg)
     {
-        Element el2 = ElementTransformer.transform(funcOp.getElement(), transform);
+        Element el2 = ElementTransformer.transform(funcOp.getElement(), transform, this);
```

Comment:
This is my actual fix attempt.

Reply:
The fix is extended with a few additional line in a separate comment. Closing this one.